src/gazebo/ros_gz_description/models/master_ASM/urdf/master_ASM.urdf

```diff
@@ -44,6 +44,74 @@
     </collision>
   </link>
 
+  
+  <joint name="camera_joint" type="fixed">
+        <parent link="base_link"/>
+        <child link="camera_link"/>
+        <origin xyz="0.35 0 0.05" rpy="0 0 0"/>
+    </joint>
+ 
+    <link name="camera_link">
+        <visual>
+            <geometry>
+                <box size="0.03025 0.17525 0.04310"/>
+            </geometry>
+            <!-- <material name="red"/> -->
+        </visual>
+    </link>
+ 
+ 
+    <!-- <joint name="camera_optical_joint" type="fixed">
+        <parent link="camera_link"/>
+        <child link="camera_link_optical"/>
+        <origin xyz="0 0 0" rpy="1.57 0 1.57"/>
+    </joint>
+ 
+    <link name="camera_link_optical"></link> -->
+ 
+ 
+ 
+    <gazebo reference="camera_link">
+ 
+        <sensor name="camera" type="camera">
+            <pose> 0 0 0 0 0 0 </pose>
+            <!-- <visualize>true</visualize> -->
+            <update_rate>30</update_rate>
+            <camera>
+                <horizontal_fov>1.91986</horizontal_fov>
+                <image>
+                    <format>R8G8B8</format>
+                    <width>1280</width>
+                    <height>720</height>
+                </image>
+                <clip>
+                    <near>0.05</near>
+                    <far>8.0</far>
+                </clip>
+            </camera>
+            <topic>camera</topic>
+        </sensor>
+
+        <sensor name="depthcamera" type="depth">

```

Comment:
The depth camera is not currently working properly, should we take it out of this PR or fix it?